storage-operator/pkg/controller/volume/volume_controller.go

```diff
@@ -88,85 +488,214 @@ type ReconcileVolume struct {
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileVolume) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	reqLogger := log.WithValues("Request.Name", request.Name)
-	reqLogger.Info("Reconciling Volume")
+	reqLogger.Info("reconciling volume: START")
```

Comment:
I would log this at Debug level or so rather than Info.

Reply:
> log.Debug undefined (type logr.Logger has no field or method Debug)

No debug method it seems, also not found in the [doc](https://godoc.org/sigs.k8s.io/controller-runtime/pkg/log)